artemis-console-extension/artemis-extension/src/artemis-extension/artemis/config-manager.ts

```diff
@@ -0,0 +1,63 @@
+import { log } from "./globals"
```

Comment:
Missing licence header failed the build on the RAT check. Related, the RAT check step only uploaded one result file, but they are generated per-module so the failure isnt actually in the uploaded file.